README.md

```diff
@@ -1 +1,56 @@
-# react-todo-app
\ No newline at end of file
+# Todo App React
+
+## A website that keeps track of activities to perform during the day and checks and deletes completed activities.
+
+## Built With
+
+- React
+
+To get a local copy up and running follow these simple example steps.
+
+1. Clone Repository 
+  
+  Run
+   ```
+   git clone git@github.com:Achilles-Dev/react-todo-app.git
+   ```
+
+### Install
+
+  Run
+   ```
+   npm install 
+   ```
+
+### Link to Online Version
+
+   [Todo App React](https://achilles-dev.github.io/react-todo-app/build) 
+
+## Authors
+
+👤 **Solomon Hagan**
+
+- GitHub: [@Achilles-Dev](https://github.com/Achilles-Dev/)
+- Twitter: [@SoloAchilles](https://twitter.com/SoloAchilles/)
+- LinkedIn: [Solomon Hagan](https://www.linkedin.com/in/solomon-hagan-b51693138/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Very good updating your readme, Please add the License file you mentioned here. You can use [Microverse MIT License Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)